server/src/modes/template/tagProviders/index.ts

```diff
@@ -76,9 +76,12 @@ export function getTagProviderSettings(workspacePath: string | null | undefined)
     if (dependencies['buefy']) {
       settings['buefy'] = true;
     }
-    if (dependencies['vuetify']) {
+    if (dependencies['vuetify'] || devDependencies['vuetify']) {
       settings['vuetify'] = true;
     }
+    if (dependencies['@nuxtjs/vuetify'] || devDependencies['@nuxtjs/vuetify']) {
+      dependencies['vuetify'] = true;
```

Comment:
Shouldn't it be :
```suggestion
      settings['vuetify'] = true;
```
?

Reply:
No. Maybe the nuxt check should happen _before_ the vuetify check so that `settings['vuetify']` gets set as well 🤷‍♂ . I am not actually 100% sure what setting the `settings` object accomplishes 😕 . 

The actual loading of the autocomplete data specified in `vetur` package.json option happens here https://github.com/vuejs/vetur/blob/4056e3d3c9c49737f1ecb17383bb8b8340089b7e/server/src/modes/template/tagProviders/index.ts#L116 and that depends on what is inside the `dependencies` object. Since vuetify is not a direct dependency when using nuxtjs module, we fake it by inserting vuetify.

Reply:
Good catch, then all good I believe